.travis.yml

```diff
@@ -178,12 +178,11 @@ jobs:
       - openssl aes-256-cbc -K $encrypted_1019fd6574a7_key -iv $encrypted_1019fd6574a7_iv -in tests/e2e/e2e-gcp-sa-key.json.enc -out tests/e2e/e2e-gcp-sa-key.json -d
       - gcloud auth activate-service-account --key-file tests/e2e/e2e-gcp-sa-key.json
       - ./build/get_ext_tools.sh
-      - ./tests/e2e/deployment-test-cleanup.sh
```

Comment:
This is a cleaning script meant for after the tests. We do have 2 other scripts at this point for the necessary pre-cleaning.